docs/reference/objects/product/index.md

```diff
@@ -164,6 +170,7 @@ boolean
 {: .label .fs-1 }
 
 Returns `true` if one of the [variants]({% link docs/reference/objects/product/variant/index.md %}) in the product has an active [promotion]({% link docs/reference/objects/product/promotion.md %}).
+If a product is a recurring experience, this returns `true` if any variant available has an active promotion.
```

Comment:
I don't think we need to change the docs for this one. At least not until we implement date-specific promotions.